webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -71,24 +61,34 @@ const CategorySelector = (props) => {
           {i18n.t('common.manage')}
         </button>
       )}
-      {showCategoriesPanel && !editedCategory && (
+      {showCategoriesPanel && !showCategoryPanel && (
```

Comment:
does this recreate the categories panel slide animation when you close the category panel?